src/main/asciidoc/index.adoc

```diff
@@ -1578,12 +1578,27 @@ Native on BSD gives you extra networking options:
 
 === Domain sockets
 
-Natives provide support domain sockets for `NetServer`:
+Natives provide domain sockets support for `NetServer` and `HttpServer`:
 
 [source,$lang]
 ----
 {@link examples.CoreExamples#serverWithDomainSockets}
+----`
+
+Or for http:
+[source,$lang]
 ----
+vertx.createHttpServer().requestHandler(req -> {
+  // Handle application
```

Comment:
can you move this to CoreExamples like serverWithDomainSockets ? this code will be translated into other languages using a code translator